tests/expected/c/02505-align_assigns.c

```diff
@@ -0,0 +1,46 @@
+unsigned foo(unsigned in1, unsigned in2, unsigned in3, unsigned in4) {
+	unsigned a     = 0xa;
+	unsigned b     = 0xb;
+	unsigned c     = 0xc;
+	unsigned x[10] = {0};
+	{
+		unsigned xyz = (
+			1
+			+
+			2
+			+
+			3
+			+
+			4
+			+
+			5
+			);
+	}
+	unsigned r1       = in1 * in3;
+	unsigned r2first  = in1 * in4;
+	unsigned xyzzy    = 123,
```

Comment:
The declaration of more than ONE variable is always a problem.
An other places this is NOT correctly supported.

Reply:
This is a test file, I think it shouldn't matter how variables are defined. In fact it could even help finding bugs in real user code usage.
For uncrustify source code, I agree on the single variable definition per line.